ramls/edge-orders.raml

```diff
@@ -10,33 +10,46 @@ types:
 /orders/validate:
   displayName: Validate
   get:
-    description: Validate that the API Key provided can be used to place an order
+    description: Validate that the API Key provided can be used to place an order. The response body depends on the Vendor (GOBI or other)
```

Comment:
it would be better to say that it depends on the type query argument in the request

Reply:
Thanks. Changed